src/components/BusinessReviews/ReviewsCard.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import LinearProgress from "@material-ui/core/LinearProgress";
+import { Rating } from "@material-ui/lab";
+
+import "./styles.scss";
+export default function ReviewsCard(props) {
```

Comment:
we are going to use this component somewhere else, so we should put it on a separate folder, called ReviewCard (notice that i removed the `s` from reviews)